scripts/variantstore/TERRA_QUICKSTART.md

```diff
@@ -92,17 +92,32 @@ This is done by running the `GvsCreateFilterSet` workflow with the following par
 
 This step extracts the data in BigQuery into a sharded joint called VCF 
 
+To prepare the dataset for extraction by putting it into a single vet and ref_ranges table, run the `GvsPrepareRangesCallset` workflow with the following parameters:
+
+
+| Parameter      | Description                                                                                     |
+| ---------------- |-------------------------------------------------------------------------------------------------|
+| data_project | The name of the google project containing the dataset                                           |
+| default_dataset  | The name of the dataset                                                                         |
+| destination_cohort_table_prefix | The name of the preparation table that this step is creating                                    |
```

Comment:
Maybe add an example to the description here?  `e.g. a table named <destination_cohort_table_prefix>__REF_RANGES will be created` just to make it clear that it's a prefix.